### PR TITLE
Update Purview catalog SDK for python

### PR DESCRIPTION
**Pull request by yifan-zhou922**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by msyyc — CHANGES_REQUESTED** [review]
(1) the `CHANGELOG.md` is not updated since you need to add the content manually
(2) `_version.py` is not updated to `1.0.0b3`, please check it
(3) Please run the test and update recordings.

File: sdk/purview/azure-purview-catalog/CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,9 @@
 
 ## 1.0.0b3 (Unreleased)
 
-### Features Added
+**Bugs Fixed**
```
Comment (msyyc) [inline sdk/purview/azure-purview-catalog/CHANGELOG.md:5]: need to update release date

Comment (yifan-zhou922) [reply sdk/purview/azure-purview-catalog/CHANGELOG.md:5]: updated. thx

**Comment by msyyc** [comment]
some content is merged by https://github.com/Azure/azure-sdk-for-python/pull/22495

**Review by msyyc — APPROVED** [review]

**Review by johanste — APPROVED** [review]

**Pull request merged**